src/components/DepositWidget/Row.tsx

```diff
@@ -75,6 +163,20 @@ export const Row: React.FC<RowProps> = (props: RowProps) => {
   const [visibleForm, showForm] = useState<'deposit' | 'withdraw' | void>()
   const exchangeBalanceTotal = exchangeBalance.add(depositingBalance)
 
+  // Checks innerWidth
+  let showResponsive = innerWidth < 500
```

Comment:
Same concern as way way below, reflow happens here.
Use your hook.

Reply:
even tho innerWidth here is a prop? (it's not from window.innerWidth)